river/preprocessing/one_hot.py

```diff
@@ -1,16 +1,18 @@
 import collections
 
 from river import base
-from typing import Dict, Union
+from collections.abc import Iterable
 
-__all__ = ["OneHotEncoder", "OneHotListEncoder"]
+__all__ = ["OneHotEncoder"]
 
 
 class OneHotEncoder(base.Transformer):
     """One-hot encoding.
 
-    This transformer will encode every feature it is provided it with. You can apply it to a
-    subset of features by composing it with `compose.Select` or `compose.SelectType`.
+    This transformer will encode every feature it is provided it with.
+    If an iterable is provded, this transformer will encode every entry in the iterable.
```

Comment:
`provided` typo

I would say "list or set" rather than "iterable" to be precise. Indeed, a `str` is an iterable.

Reply:
Removed the extra 'it' and replaced Iterable with list/set